packaging/installer/methods/kickstart-64.md

```diff
@@ -0,0 +1,82 @@
+> [nightly vs. stable releases](../README.md#nightly-vs-stable-releases).
+
+## What does `kickstart-static64.sh` do?
+
+The `kickstart.sh` script does the following after being downloaded and run:
+
+-   Detects the Linux distribution and **installs the required system packages** for building Netdata. Unless you added
+    the `--dont-wait` option, it will ask for your permission first.
+-   Downloads the latest Netdata binary from the [binary-packages](https://github.com/netdata/binary-packages)
+    repository. You can also run any of these `.run` files with [makeself](https://github.com/megastep/makeself).
+-   Installs Netdata by running `./netdata-installer.sh` from the source tree, including any options you might have
+    added.
+-   Installs `netdata-updater.sh` to `cron.daily` to enable automatic updates, unless you added the `--no-updates`
+    option.
+-   Prints a message about whether the insallation succeeded for failed for QA purposes.
+
+If your shell fails to handle the above one-liner, you can download and run the `kickstart-static64.sh` script manually.
+
+```sh
+# download the script with curl
+curl https://my-netdata.io/kickstart-static64.sh >/tmp/kickstart-static64.sh
+
+# or, download the script with wget
+wget -O /tmp/kickstart-static64.sh https://my-netdata.io/kickstart-static64.sh
+
+# run the downloaded script (any sh is fine, no need for bash)
+sh /tmp/kickstart-static64.sh
+```
+
+## Optional parameters to alter your installation
+
+The `kickstart-static64.sh` script passes all its parameters to `netdata-installer.sh`, which you can use to customize
+your installation. Here are a few important parameters:
+
+-   `--dont-wait`: Enable automated installs by not prompting for permission to install any required packages.
+-   `--dont-start-it`: Prevent the installer from starting Netdata automatically.
+-   `--stable-channel`: Automatically update only on the release of new major versions.
+-   `--nightly-channel`: Automatically update on every new nightly build.
+-   `--disable telemetry`: Opt-out of [anonymous statistics](../../../docs/anonymous-statistics.md) we use to make
+    Netdata better.
+-   `--no-updates`: Prevent automatic updates of any kind.
+-   `--local-files`: Used for [offline installations](offline.md). Pass four file paths: the Netdata
+    tarball, the checksum file, the go.d plugin tarball, and the go.d plugin config tarball, to force kickstart run the
+    process using those files. This option conflicts with the `--stable-channel` option. If you set this _and_
+    `--stable-channel`, Netdata will use the local files.
+
+## Verify script integrity
+
+To use `md5sum` to verify the intregity of the `kickstart-static64.sh` script you will download using the one-line
+command above, run the following:
+
+```bash
+[ "23e0f38dfb9d517be16393c3ed1f88bd" = "$(curl -Ss https://my-netdata.io/kickstart-static64.sh | md5sum | cut -d ' ' -f 1)" ] && echo "OK, VALID" || echo "FAILED, INVALID"
```

Comment:
Because of this being moved, we need updates to CI (we check the hashes here to make sure they match the most recent state of the repository). This is why Tavis CI is failing right now on this PR.

If desired, I can push a commit to the PR branch to take care of this.

Reply:
I'd appreciate that in this case. Maybe I can learn how to do this myself in a smaller, more contained PR.

Reply:
I'd be happy to provide instructions if you would like to do it yourself, it's not particularly hard to do (should be no more than two lines changed in a single script, and it's only updating file names), I just figured you might want to not have to worry about it right now.

Reply:
Correction, after looking at the relevant code again, it's a bit more involved than I thought (there are structural changes needed, since it assumes both the things we're checking checksums for have the checksums listed in the same file).

I'll make sure to document in the commit what needs to be updated if these get moved again in the future.

Reply:
Commit added to update the CI stuff.

For future reference, any further changes that need to be made because of moving these lines just need to modify the lines in `tests/installer/checksums.sh` near the end that start with `check_checksums` to point to the correct files for the respective kickstart scripts.